### DISCUSSION TITLE
Welcome to Script Winter Of Code

### DISCUSSION THREAD

**Discussion by vinitshahdeo** [vinitshahdeo/PortScanner#114] [General]
:wave: Hello,

**Thanks for showing your interest in [SWoC](https://swoc.tech/).** 
Super glad to know that you're interested in contributing to **[Port Scanner](https://github.com/vinitshahdeo/PortScanner/)** project.

Feel free to ask your doubts here. Kindly note that the community bonding period ends on 4<sup>th</sup> Jan. I recommend you to go through the [open issues](https://github.com/vinitshahdeo/PortScanner/issues/) and have an idea prior to the event. There will be more **first-timers** issues opened by us soon.

Before we start, I highly recommend all of you to go through the following medium articles:

- [GO-GREEN featuring GITHUB](https://medium.com/@kashish_121/go-green-featuring-github-f8750fbf0729)
- [git push: You just need a little PUSH](https://medium.com/@ishikadubey2000/look-mom-im-on-github-521bb6c1f51d)

> Keep watching this repo! 

_Regards,_
**[Vinit Shahdeo](https://www.linkedin.com/in/vinitshahdeo/)**
**Project Admin**

[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/vinitshahdeo)

**Comment by Ritu01-git** [comment]
In this project we have Python 2 but This version is not supported now days. What to do? install python 2 or do anything else?

**Reply by vinitshahdeo** [reply]
Hey @Ritu01-git,

We maintain a separate branch for Python 3. You can check out the `README.md` for more details.
